sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -72,6 +72,20 @@
     private static final int DEFAULT_QUERY_PLAN_RESPONSE_TIMEOUT_IN_SECONDS = 5;
     private static final int DEFAULT_ADDRESS_REFRESH_RESPONSE_TIMEOUT_IN_SECONDS = 5;
 
+    // SessionTokenMismatchRetryPolicy Constants
+    // TODO fabianm - do we really want to keep these config overrides like in .Net
+    private static final String DEFAULT_SESSION_TOKEN_MISMATCH_WAIT_TIME_IN_MILLISECONDS_NAME =
+        "session-token-mismatch-policy-default-wait-time-milliseconds";
+    private static final int DEFAULT_SESSION_TOKEN_MISMATCH_WAIT_TIME_IN_MILLISECONDS = 5000;
+
+    private static final String DEFAULT_SESSION_TOKEN_MISMATCH_INITIAL_BACKOFF_TIME_IN_MILLISECONDS_NAME =
+        "session-token-mismatch-policy-default-initial-backoff-milliseconds";
+    private static final int DEFAULT_SESSION_TOKEN_MISMATCH_INITIAL_BACKOFF_TIME_IN_MILLISECONDS = 5;
+
+    private static final String DEFAULT_SESSION_TOKEN_MISMATCH_MAXIMUM_BACKOFF_TIME_IN_MILLISECONDS_NAME =
+        "session-token-mismatch-policy-default-maximum-backoff-milliseconds";
```

Comment:
the new config names not consistent with the existing ones.
```suggestion
        "COSMOS.SESSION_TOKEN_MISMATCH_POLICY_DEFAULT_MAX_BACKOF_IN_MILLISECONDS"
```

same for the other configs

Reply:
Fixed in next iteration